pkgs/applications/misc/bottles/default.nix

```diff
@@ -95,6 +95,11 @@ python3Packages.buildPythonApplication rec {
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
   '';
 
+  passthru.updateScript = gitUpdater {
+    inherit pname version;
+    url = "https://github.com/bottlesdevs/Bottles.git";
```

Comment:
This ill produce next url `https://github.com/bottlesdevs/bottles/archive/2022.5.28-trento-3.tar.gz` instead of expected, because as default fetchFromGithub using fetchzip and useFetchGit (see next lines) not enabled
https://github.com/NixOS/nixpkgs/blob/385a38d1f6623797ba912c1e3d0dcfd2f965c09b/pkgs/build-support/fetchgithub/default.nix#L27-L30
https://github.com/NixOS/nixpkgs/blob/385a38d1f6623797ba912c1e3d0dcfd2f965c09b/pkgs/build-support/fetchgithub/default.nix#L48-L53
And I don't sure we should change this

Reply:
<details>

<summary>`nix eval -f . --json bottles.src.drvAttrs | jq .`</summary>

``` json
{
  "SSL_CERT_FILE": "/no-cert-file.crt",
  "__ignoreNulls": true,
  "args": [
    "-e",
    "/nix/store/lphxcbw5wqsjskipaw1fb8lcf6pm6ri6-builder.sh"
  ],
  "buildInputs": [],
  "builder": "/nix/store/3j18grljsyy4nxc078g00sy4cx6cf16g-bash-5.1-p16/bin/bash",
  "configureFlags": [],
  "curlOpts": "",
  "curlOptsList": "",
  "depsBuildBuild": [],
  "depsBuildBuildPropagated": [],
  "depsBuildTarget": [],
  "depsBuildTargetPropagated": [],
  "depsHostHost": [],
  "depsHostHostPropagated": [],
  "depsTargetTarget": [],
  "depsTargetTargetPropagated": [],
  "doCheck": false,
  "doInstallCheck": false,
  "downloadToTemp": true,
  "executable": false,
  "impureEnvVars": [
    "http_proxy",
    "https_proxy",
    "ftp_proxy",
    "all_proxy",
    "no_proxy",
    "NIX_CURL_FLAGS",
    "NIX_HASHED_MIRRORS",
    "NIX_CONNECT_TIMEOUT",
    "NIX_MIRRORS_alsa",
    "NIX_MIRRORS_apache",
    "NIX_MIRRORS_bioc",
    "NIX_MIRRORS_bitlbee",
    "NIX_MIRRORS_centos",
    "NIX_MIRRORS_cpan",
    "NIX_MIRRORS_debian",
    "NIX_MIRRORS_fedora",
    "NIX_MIRRORS_gcc",
    "NIX_MIRRORS_gentoo",
    "NIX_MIRRORS_gnome",
    "NIX_MIRRORS_gnu",
    "NIX_MIRRORS_gnupg",
    "NIX_MIRRORS_hackage",
    "NIX_MIRRORS_hashedMirrors",
    "NIX_MIRRORS_ibiblioPubLinux",
    "NIX_MIRRORS_imagemagick",
    "NIX_MIRRORS_kde",
    "NIX_MIRRORS_kernel",
    "NIX_MIRRORS_luarocks",
    "NIX_MIRRORS_maven",
    "NIX_MIRRORS_mozilla",
    "NIX_MIRRORS_mysql",
    "NIX_MIRRORS_openbsd",
    "NIX_MIRRORS_opensuse",
    "NIX_MIRRORS_osdn",
    "NIX_MIRRORS_postgresql",
    "NIX_MIRRORS_pypi",
    "NIX_MIRRORS_qt",
    "NIX_MIRRORS_roy",
    "NIX_MIRRORS_sageupstream",
    "NIX_MIRRORS_samba",
    "NIX_MIRRORS_savannah",
    "NIX_MIRRORS_sourceforge",
    "NIX_MIRRORS_steamrt",
    "NIX_MIRRORS_tcsh",
    "NIX_MIRRORS_testpypi",
    "NIX_MIRRORS_ubuntu",
    "NIX_MIRRORS_xfce",
    "NIX_MIRRORS_xorg"
  ],
  "mirrorsFile": "/nix/store/7s4lad3h36fry9iblbxmqdwiv7zlky4l-mirrors-list",
  "name": "source",
  "nativeBuildInputs": [
    "/nix/store/4f6fm122zp0nwjc74pnvy6s2y8z4nkwm-curl-7.83.1-dev",
    "/nix/store/bwbwx0kc1c1hz5l97dr124fd62aswc22-unzip-6.0",
    "/nix/store/53nfnxv2qf02jl48bgxabk8wlc3ivmhx-glibc-locales-2.34-210"
  ],
  "nixpkgsVersion": "22.11",
  "outputHash": "sha256-KIDLRqDLFTsVAczRpTchnUtKJfVHqbYzf8MhIR5UdYY=",
  "outputHashAlgo": "sha256",
  "outputHashMode": "recursive",
  "outputs": [
    "out"
  ],
  "patches": [],
  "postFetch": "unpackDir=\"$TMPDIR/unpack\"\nmkdir \"$unpackDir\"\ncd \"$unpackDir\"\n\nrenamed=\"$TMPDIR/2022.5.28-trento-3.tar.gz\"\nmv \"$downloadedFile\" \"$renamed\"\nunpackFile \"$renamed\"\nchmod -R +w \"$unpackDir\"\nif [ $(ls \"$unpackDir\" | wc -l) != 1 ]; then\n  echo \"error: zip file must contain a single file or directory.\"\n  echo \"hint: Pass stripRoot=false; to fetchzip to assume flat list of files.\"\n  exit 1\nfi\nfn=$(cd \"$unpackDir\" && echo *)\nif [ -f \"$unpackDir/$fn\" ]; then\n  mkdir $out\nfi\nmv \"$unpackDir/$fn\" \"$out\"\n\n\nchmod 755 \"$out\"\n",
  "postHook": null,
  "preferHashedMirrors": true,
  "preferLocalBuild": true,
  "propagatedBuildInputs": [],
  "propagatedNativeBuildInputs": [],
  "showURLs": false,
  "stdenv": "/nix/store/1hb1k600qhl39z7xp3nlflwir3jqnsz4-stdenv-linux",
  "strictDeps": false,
  "system": "x86_64-linux",
  "urls": [
    "https://github.com/bottlesdevs/bottles/archive/2022.5.28-trento-3.tar.gz"
  ],
  "userHook": null
}

```

</details>

Reply:
bad guess. We want to use the following:

```suggestion
    inherit (src) gitRepoUrl;
```

Reply:
We can't: src derivation don't provide it 
Look in message in spoiler above, I've print this derivation arguments 

Explanation: into last line of fetchFromGitHub only fetcherArgs pass into fetcher 
However, you can see passthru.gitRepoUrl passed into fetcher, but into fetcher itself passthru will ignored

Reply:
It is used in the unstable updater, so that needs to work https://github.com/NixOS/nixpkgs/blob/dc7513c64078aaeef5842ce74585dc9d7bc08b4d/pkgs/common-updater/unstable-updater.nix#L52

Reply:
It seems, we can just delete this string, and it still works
Look into current code